tests/common/helpers/parallel.py

```diff
@@ -77,3 +77,29 @@ def parallel_run(target, args, kwargs, nodes, timeout=None):
     logger.info('Completed running processes for target "{}" in {} seconds'.format(target.__name__, str(delta_time)))
 
     return results
+
+
+def reset_ansible_local_tmp(target):
+    """Decorator for resetting ansible default local tmp dir for parallel multiprocessing.Process
+
+    Args:
+        target (function): The function to be decorated.
+    """
+
+    def wrapper(*args, **kwargs):
+
+        # Reset the ansible default local tmp directory for the current subprocess
+        # Otherwise, multiple processes could share a same ansible default tmp directory and there could be conflicts
+        from ansible import constants
+        import os, shutil, tempfile
```

Comment:
maybe move imports to the top of the file with other imports? os was imported before and LGTM is complaining about that.

Reply:
Moved `import os, shutil, tempfile` to top of file. Since importing ansible is just for updating temp folder for sub-processes, this importing is still kept here.